src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -38,29 +46,56 @@ class ProductContext implements Context
 
     /**
      * @param RepositoryInterface $productRepository
+     * @param RepositoryInterface $taxCategoryRepository
      * @param SharedStorageInterface $sharedStorage
      * @param FactoryInterface $productFactory
      */
-    public function __construct(RepositoryInterface $productRepository, SharedStorageInterface $sharedStorage, FactoryInterface $productFactory)
-    {
+    public function __construct(
+        RepositoryInterface $productRepository,
+        RepositoryInterface $taxCategoryRepository,
+        SharedStorageInterface $sharedStorage,
```

Comment:
I am not sure it will be always injected but yeah, if it is, let's make it the first argument.